packages/@okta/vuepress-site/guides/sign-into-web-app/require-authentication/aspnetcore/reqauthspecific.md

```diff
@@ -1,4 +1,4 @@
-Use the `Authorize` attribute on controllers or actions to require a logged-in user:
+Use the `[Authorize]` attribute on controllers or actions to require a logged-in user:
```

Comment:
same

Use the `[Authorize]` attribute on controllers or actions to require a signed-in user: